lib/devise_saml_authenticatable/saml_response.rb

```diff
@@ -0,0 +1,16 @@
+require 'devise_saml_authenticatable/saml_mapped_attributes'
+
+module SamlAuthenticatable
+  class SamlResponse
```

Comment:
Is this class necessary? It seems like a simple factory for `SamlMappedAttributes`, which doesn't seem like it's needed since the constructor is so straightforward.